src/ScriptCs.Hosting/ScriptServicesBuilder.cs

```diff
@@ -131,6 +133,18 @@ public IScriptServicesBuilder Debug(bool debug = true)
             return this;
         }
 
+        public IScriptServicesBuilder SetOverride<TContract, TImpl>(TImpl value) where TImpl : TContract
+        {
+            Overrides[typeof(TContract)] = value;
+            return this;
+        }
+
+        public IScriptServicesBuilder SetOverride<TContract, TImpl>() where TImpl : TContract
+        {
+            Overrides[typeof(TContract)] = typeof(TImpl);
```

Comment:
Ah yes, I see it now... https://github.com/scriptcs/scriptcs/blob/9a3b4b6f1d83086a5558b874d6b265cf2a3ffa6f/src/ScriptCs.Hosting/ScriptServicesRegistration.cs#L31-L38

Thanks!